tests/integration/test_vulnerability_detector/test_feeds/canonical/test_invalid_syntax_canonical_feed.py

```diff
@@ -18,34 +17,31 @@
 pytestmark = pytest.mark.tier(level=2)
 
 current_test_path = os.path.dirname(os.path.realpath(__file__))
-test_data_path = os.path.join(current_test_path, '..', 'data')
-configurations_path = os.path.join(test_data_path, 'configuration', vd.INVALID_CANONICAL_FEEDS_CONF)
+test_data_path = os.path.join(current_test_path, '..', '..', 'data')
+configurations_path = os.path.join(test_data_path, 'configuration', 'test_feeds', vd.INVALID_CANONICAL_FEEDS_CONF)
```

Comment:
Same as before. It would be interesting to move them to `vulnerability_detector` module.